tests/e2e/specs/in-game/one-offs/targeting_cleanup.spec.js

```diff
@@ -168,7 +165,6 @@ describe('Clean-up of One-Off Targets', () => {
   });
 
   it('ONE-OFF Target should be removed after one-off is COUNTERED - target is POINTS', () => {
-    cy.skipOnGameStateApi();
```

Comment:
This one *might* be ready because the test counters the 9's effect and countering has been built (unlike resolving the 9). We can unskip it like you have here and if it fails we can just skip it